scripts/jest/setupTestFrameworkScriptFile.js

```diff
@@ -0,0 +1,8 @@
+// Any code added here will run before each jest suite is run
+
+const enzyme = require('enzyme')
+const Adapter = require('enzyme-adapter-react-16')
+
+enzyme.configure({ adapter: new Adapter() })
+
+// require('jest-styled-components')
```

Comment:
Add tests for some emotion styled components and determine if something like this is necessary.